opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
I agree that this doesn't need to be defined, and defining it is distracting at this point. When an OpenTracing implementation is realized against a real system, I'm sure a subtype will do something like this in their own way that makes sense for them. Ex cloud trace [special-cases http codec](https://github.com/GoogleCloudPlatform/cloud-trace-java/blob/master/cloud-trace-sdk-java-core/src/main/java/com/google/cloud/trace/sdk/TraceContext.java#L53), and I expect future finagle things to defer to their context propagation api.